File: src/main/kotlin/com/example/kotlin/dal/entity/CreditReportEntity.java
package com.example.kotlin.dal.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
@Getter
@Setter
@AllArgsConstructor
public class CreditReportEntity {

    @Id
    private id: Long
}

File: src/main/kotlin/com/example/kotlin/dal/entity/CreditReportEntity.kt
package com.example.kotlin.dal.entity

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.Setter
import org.hibernate.annotations.CreationTimestamp
import org.hibernate.annotations.Type
import org.hibernate.annotations.UpdateTimestamp
import java.time.LocalDateTime
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity
@Getter
@Setter
@AllArgsConstructor
class CreditReportEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    val id: Long,

    val inn: String,
    val creditInfoId: Int,

    @Type(type = "text")
    val creditReportZip: String,

    @CreationTimestamp
    val createdAt: LocalDateTime,

    @UpdateTimestamp
    val updatedAt: LocalDateTime
)